### PR TITLE
Update task list page tags and warning text

### PR DESCRIPTION
**Pull request by mforner13**
This PR addresses feedback received from user research on the task list page. 

It changes the following:

1. Changes the 'Completed' status of a monthly update to be 'Ready to submit'
2. Updates the content of the warning text on the task list, which now shows `x out of n actions are not ready to be submitted` when there are outstanding updates or `All updates are ready to be submitted`. 

Due to the 'Ready to submit' label being wider than 'Completed', I adjusted the width of the left table cell in the task list, so that the label was all on one line. Because the width of a table cell automatically applies to the whole column, I moved the table header out into a caption rather than being in a row, so that the text can all be on one line, as it shows in the design.

I also made the following changes so that the task list reflects the current designs:

1. Made the submitted tag green
2. Right aligned the task list tags

### Ready to submit tags & incomplete warning text

Screenshot 2021-05-25 at 12 09 25

### Complete warning text
Screenshot 2021-05-25 at 12 10 26

### Green submitted tag
image

### CODE REVIEW

**Review by NickFitz — APPROVED** [review]
Looks good 👍

File: defend_data_capture/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -37,6 +37,14 @@ $govuk-image-url-function: frontend-image-url;
   background-color: govuk-colour("light-grey", $legacy: "grey-4");
 }
 
+.app-right-aligned-tag {
```
Comment (dev-digital) [inline defend_data_capture/assets/stylesheets/application.scss:40]: There is a css file for supply chain app, may be we could move it there

**Review by dev-digital — APPROVED** [review]
I liked the screen shots :+1:

File: defend_data_capture/assets/stylesheets/application.scss (hunk 1 above)
Comment (mforner13) [reply defend_data_capture/assets/stylesheets/application.scss]: Good thinking - have added to the supply chain css file :)

**Pull request merged**